src/azure-cli/azure/cli/command_modules/ams/tests/latest/test_ams_account_encryption_scenarios.py

```diff
@@ -9,7 +9,7 @@
 class AmsEncryptionTests(ScenarioTest):
     @ResourceGroupPreparer(location='centralus')
     @StorageAccountPreparer(parameter_name='storage_account_for_create')
-    @KeyVaultPreparer(location='centralus')
+    @KeyVaultPreparer(location='centralus', additional_params='--enable-purge-protection')
```

Comment:
The refactoring of `KeyVaultPreparer` could be split into another PR.

Reply:
It's part of re-recording creating keyvault in other modules.